p8e-proto-internal/src/main/proto/p8e/common.proto

```diff
@@ -17,6 +17,8 @@ message Signature {
 
     // Identity of signer
     SigningAndEncryptionPublicKeys signer = 4;
+
+    UUID key_id = 5;
```

Comment:
SmartKey has a key_id as part of the signature that we need we need to verify the signature.

Reply:
Can we get away with not having this? When you upload or create a new key in smart key it lets you download the public key. We can store that public key the same way we do in the affiliates table now. It would just also have the smart_key_uuid on the row as well.

Reply:
Yea, I could make the change along with this ticket ... [https://app.clubhouse.io/figure/story/88583/update-affiliate-table-to-include-securityobject-table-uuid](Update affiliate table to include SecurityObject Table UUID), I've made a note to remove it.